src/messages/NetworkMessage.h

```diff
@@ -0,0 +1,312 @@
+/*
+ * This file is part of the bitcoin-classic project
+ * Copyright (c) 2009-2010 Satoshi Nakamoto
+ * Copyright (c) 2009-2015 The Bitcoin Core developers
+ * Copyright (C) 2017 Tom Zander <tomz@freedommail.ch>
+ * Copyright (C) 2017 Angel Leon <@gubatron>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef BITCOINCLASSIC_NETWORKMESSAGE_H
+#define BITCOINCLASSIC_NETWORKMESSAGE_H
+
+#include <main.h>
+#include <consensus/validation.h>
+#include <net.h>
+#include <chainparams.h>
+#include <timedata.h>
+#include <addrman.h>
+#include <txorphancache.h>
+#include <txmempool.h>
+#include <merkleblock.h>
+#include <thinblock.h>
+#include <validationinterface.h>
+
+
+static bool CheckIndexAgainstCheckpoint(const CBlockIndex* pindexPrev, CValidationState& state, const CChainParams& chainparams, const uint256& hash);
+
+namespace {
```

Comment:
👍

Reply:
Quick question:

In AdminServer.cpp, they prefix every method with "Admin::" which is the namespace name.

Is it ok in my case to simply name the namespace, as all my .cpp's are already enclosed in

`namespace {`

so I'd do:

`namespace Network {`

instead of adding:

`Network::` as a prefix for every method and removing the existing `namespace {` at the top of the .cpps